src/algebra/order/monoid_lemmas_zero_lt.lean

```diff
@@ -479,8 +553,8 @@ variables [mul_one_class α] [has_zero α]
 section preorder
 variables [preorder α]
 
--- Lemmas in the form of `a ≤ a * b ↔ 1 ≤ b` and `a * b ≤ a ↔ b ≤ 1`,
--- which assume left covariance.
+/-! Lemmas in the form of `a ≤ a * b ↔ 1 ≤ b` and `a * b ≤ a ↔ b ≤ 1`,
```

Comment:
```suggestion
/-! Lemmas of the form `a ≤ a * b ↔ 1 ≤ b` and `a * b ≤ a ↔ b ≤ 1`,
```
Just a typo.  Same suggestion for the remaining comments converted to doc-strings.